.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
     - env: MODE=semantic PR_ONLY=false
     - env: MODE=model PR_ONLY=false
     - env: MODE=linter PR_ONLY=true
-    - env: MODE=semantic PR_ONLY=true
+    # - env: MODE=semantic PR_ONLY=true
```

Comment:
I'd prefer to just delete this line instead of commenting it out. We don't want to accidentally have it present in both categories.